Sprint-2/interpret/invert.js

```diff
@@ -17,13 +17,33 @@ function invert(obj) {
 }
 
 // a) What is the current return value when invert is called with { a : 1 }
+//answer:{ key: 1 }
+
 
 // b) What is the current return value when invert is called with { a: 1, b: 2 }
+//answer:{ key: 2 }
 
 // c) What is the target return value when invert is called with {a : 1, b: 2}
+//answer:{ "1": "a", "2": "b" }
+
 
 // c) What does Object.entries return? Why is it needed in this program?
 
+//Object.entries(obj) returns an array of [key, value] pairs from the object:
+
 // d) Explain why the current return value is different from the target output
+//In the loop, the code is using invertedObj.key = value instead of setting the value dynamically with invertedObj[value] = key. 
```

Comment:
These are all really clear explanations, good job!

Reply:
thank you for reviewing my code